bokehjs/src/lib/models/widgets/tables/cell_formatters.ts

```diff
@@ -2,14 +2,15 @@ import tz from "timezone"
 import * as Numbro from "@bokeh/numbro"
 import {_} from "underscore.template"
 
-import type * as p from "core/properties"
+import * as p from "core/properties"
 import {div, i} from "core/dom"
-import type {Color} from "core/types"
-import {FontStyle, TextAlign, RoundingFunction} from "core/enums"
+import {isField, isValue} from "core/vectorization"
+import {RoundingFunction} from "core/enums"
 import {isNumber, isString} from "core/util/types"
 import {to_fixed} from "core/util/string"
-import {color2css} from "core/util/color"
+import {color2css, rgba2css} from "core/util/color"
 import {Model} from "../../../model"
+import {ColorMapper} from "api"
```

Comment:
Generally speaking, nothing inside the library proper should use or import from `api`, it is only for external users.   So this should be updated to use the in-library location.